spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -474,6 +519,49 @@ def start_install(self):
 
         subprocess.Popen(' '.join(cmd), shell=True)
 
+    def _start_updater(self):
+        """Start updater application."""
+        info = {
```

Comment:
```suggestion
       if self.get_conf('high_dpi_custom_scale_factor', section='main'):
            scale_factors = self.get_conf(
                'high_dpi_custom_scale_factors',
                section='main'
            )
            scale_factor = float(scale_factors.split(":")[0])
        else:
            scale_factor = 1

        info = {
```

We need to get the scale factor from our config system.

Reply:
Isn't this the same as `QT_SCALE_FACTOR`?
https://github.com/spyder-ide/spyder/blob/dc00617c5504cec13348631ef5a30617a25d4cc4/spyder/app/start.py#L171-L180

Reply:
Oh, is that supposed to be `factors.split(';')` or `factors.split(':')`? `;` is in our `start.py`...

Reply:
I think we use `;` as a separator.